### PR TITLE
Merge options: ignore frames concealed and/or not at normal speed

### PR DESCRIPTION
**Pull request by JeromeMartinez**

### CODE REVIEW

File: Source/CLI/CLI_Help.cpp [hunk 1]
```diff
@@ -66,6 +66,28 @@ return_value Help(ostream& Out, const char* Name, bool Full)
         "    --merge-log value\n"
         "        Store merge log to value (file name).\n"
         "\n"
+        "    --merge-output-speed value\n"
```
Comment (dericed) [inline Source/CLI/CLI_Help.cpp:69]: What is `value`? From code, this seems to be a flag rather than needing a value.

**Comment by dericed** [comment]
@JeromeMartinez Link to a test build?

**Comment by JeromeMartinez** [comment]
> Link to a test build?

https://mediaarea.net/download/snapshots/binary/dvrescue/20220915/

note: it also has the latest merged PRs (GUI too).

**Comment by JeromeMartinez** [comment]
> I think the log should have some indication of the flags used

Updated.
No indication of flags used, but info of frames discarded when flags used (and any frame actually discarded).

```
[...]
Input files summary:
File 0:   2432278 =  84.5863% of blocks have errors, in   1658 =  86.4893% of frames

Result:
          2432278 =  84.5863% of blocks have errors, in   1658 =  86.4893% of frames

Frames discarded in output:
              369 =  19.2488% of frames discarded for non-standard playback speed.
                1 =   0.0522% of frames discarded for full concealment.
```

**Pull request merged**